### DISCUSSION TITLE
Serial println crashes using Serial print("...\n") does not

### DISCUSSION THREAD

**Discussion by xRadx** [stm32duino/Arduino_Core_STM32#2267] [General]
Just what I noticed - using Serial.println very often causes program to hang, but when at the same place I use Serial.print("...\n") the code runs w/o any problem...

Not sure what might be the problem, but it seems that the implementation of Serial println causes a problem.

**Comment by fpistm** [comment]
Could you be more precise? board used ? Is USB Serial enabled and supersed U(S)ART?....

**Reply by xRadx** [reply]
The board is Nucleo F746ZG. I am using default config for Serial. I have quite much expirience with Arduino IDE for AVRs, but I am just starting with stm32duino.

**Reply by fpistm** [reply]
Seems you open several discussion and your setup is not very clear. Are you used FreeRTOS here too?
Sketch example? ....
